src/scripts/main.js

```diff
@@ -358,3 +358,22 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const centery = Math.ceil(person.died / 100);
```

Comment:
There is a typo in the variable name `centery`. It should be `century` to correctly represent the concept of a century.